kousa/.envrc

```diff
@@ -1,9 +1,14 @@
-export GITHUB_CLIENT_ID=nyahnyahnyah
-export GITHUB_CLIENT_SECRET=foobarbaz
-
-export BEN_GITHUB_ID=nopenope
-export ACCESS_TOKEN_SECRET=oneuthateuhathoeuthuaonehntoauhaeuheae
-export REFRESH_TOKEN_SECRET=zzaneuthaonueheaouhnaoehunstasenuthanu
-
-export TWITTER_API_KEY=ntauhnaehuntaeeuntehnatu
-export TWITTER_SECRET_KEY=taeounaheunaoehutaheueteee
+export DATABASE_URL=postgres://postgres:postgres@localhost/kousa_repo2
+export BEN_GITHUB_ID=7872329
+export RABBITMQ_URL=amqp://user:password@yourinternalip:5672
+export ACCESS_TOKEN_SECRET=
+export REFRESH_TOKEN_SECRET=
+export GITHUB_CLIENT_ID=efb151849cf1c786414e
+export TWITTER_API_KEY=
+export TWITTER_SECRET_KEY=
+export TWITTER_BEARER_TOKEN=
+export GITHUB_CLIENT_SECRET=ff64d1ff2ee89fcbf8f389e0f2b9de5ae1de849a
```

Comment:
I don't think mentioning your `client-id`  and `client-secret` would be a good idea in here...
Also i don't think including this file itself would be a good idea.